live/src/main/scala/org/dbpedia/extraction/live/LiveExtractionConfigLoader.scala

```diff
@@ -90,7 +90,7 @@ object LiveExtractionConfigLoader
     val lang = Language.apply(landCode)
     val articlesSource : Source =
       if (item.getXML.isEmpty)
-        WikiSource.fromPageIDs(List(item.getItemID), new URL(apiURL), lang)
+        WikiSource.fromRevisionIDs(List(item.getItemID), new URL(apiURL), lang)
```

Comment:
I don't know for that, because in the rcStream we have only the edit id, old revision id and the new revision id.